packaging/windows/OpenRA.nsi

```diff
@@ -201,6 +202,7 @@ Function ${UN}Clean
 	Delete $INSTDIR\CHANGELOG.html
 	Delete $INSTDIR\CONTRIBUTING.html
 	Delete $INSTDIR\DOCUMENTATION.html
+	Delete $INSTDIR\Settings.html

```

Comment:
While you're at it, could you add the Lua-API.html file here as well? It seems to be missing.